Runtime/Components/CinemachineFramingTransposer.cs

```diff
@@ -422,7 +422,7 @@ public override void MutateCameraState(ref CameraState curState, float deltaTime
 
             // Compute group bounds and adjust follow target for group framing
             ICinemachineTargetGroup group = AbstractFollowTargetGroup;
-            bool isGroupFraming = group != null && m_GroupFramingMode != FramingMode.None;
+            bool isGroupFraming = group != null && !group.IsEmpty && m_GroupFramingMode != FramingMode.None;
```

Comment:
Is "IsEmpty" enough?  What if you have a group with a single member that has 0 radius?  I think you will get the same drift. Try checking for group.BoundingSphere.radius == 0 instead.

Reply:
No drifting there.
I checked empty list, list with nothing in it (None), list with 1 item zero, and non zero radius
list with more items 0 and non zero radius.

Reply:
The only time I had drifting when the group was empty